src/app/view/components/container-selector/container-selector.component.scss

```diff
@@ -0,0 +1,54 @@
+@import "variables";
+@import "mixins";
+
+.container-selector {
+    height: 206px; 
+    border: 1px solid $gray-lighter; 
+}
+
+.head-container {
+    height: 48px; 
+    padding: $basic-padding $basic-padding * 2; 
+    background-color: $gray-lighter; 
+
+    .ui-button {
+        background: transparent;
+    }
+}
+
+.head-container__close-containers {
+    float: right;
+    padding-top: $basic-padding;
```

Comment:
Use flexbox... so you can dynamically vertical center this no matter what the height is... and use flexbox always if you can.